afni_proc.py: No match.
